### DISCUSSION TITLE
zprofile not found

### DISCUSSION THREAD

**Discussion by RupakO4** [Homebrew/discussions#3254] [Getting started]
I am new to macOS and terminal in general
installing homebrew , works only until the terminal is open 
after reopening the terminal 
it shows user/.../zprofile : not found 
and brew command doesn't work
<img width="471" alt="Screen Shot 2022-05-10 at 11 08 32" src="https://user-images.githubusercontent.com/47048146/167548225-70fad10c-0792-419f-98bd-4034b49767c8.png">

**Comment by carlocab** [comment]
What’s the output of `cat ~/.zprofile`?

As an aside:

Please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse (e.g. when looking up error messages). Instead, copy the text here and wrap them in three back-ticks (```) on the line before and on the line after to format them properly.

**Reply by RupakO4** [reply]
```
# Setting PATH for Python 3.10
# The original version is saved in .zprofile.pysave
PATH="/Library/Frameworks/Python.framework/Versions/3.10/bin:${PATH}"
export PATH
==> Checking for `sudo` access (which may request your password)...
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
eval "$(/opt/homebrew/bin/brew shellenv)"
```

**Reply by carlocab** [reply]
You can remove everything there but keep one
```
eval "$(/opt/homebrew/bin/brew shellenv)"
```

You can edit this file in TextEdit using
```
open ~/.zprofile
```

**Reply by RupakO4** [reply]
should i reinstall after that??
still not working

**Reply by SMillerDev** [reply]
You should re-open the terminal app

**Reply by RupakO4** [reply]
already did that
didnt work
same error

**Reply by SMillerDev** [reply]
What does `cat ~/.zprofile` say now?

**Reply by RupakO4** [reply]
```
# Setting PATH for Python 3.10
# The original version is saved in .zprofile.pysave
PATH="/Library/Frameworks/Python.framework/Versions/3.10/bin:${PATH}"
export PATH
==> Checking for `sudo` access (which may request your password)...
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
eval "$(/opt/homebrew/bin/brew shellenv)"
```

**Reply by carlocab** [reply]
You didn’t remove everything. The only thing you should see when doing `cat ~/.zprofile` should be
```
eval "$(/opt/homebrew/bin/brew shellenv)"
```
The lines before it should not be there. You can keep the `PATH` setting if you like.

**Reply by RupakO4** [reply]
oh my bad
thanks for the help
its working now

**Reply by RupakO4** [reply]
do u know why other things were there in zprofile?
